AUTHORS

```diff
@@ -0,0 +1,3 @@
+Thomas Oster
+
+Google Inc.
```

Comment:
So you are contributing while working? Or do you have to add this reference even for private work? I am just curious why Google is interested in VisiCut.